packages/ember-render-modifiers/dist/modifiers/did-insert.js

```diff
@@ -0,0 +1,63 @@
+import { setModifierManager, capabilities } from '@ember/modifier';
```

Comment:
@ctjhoa dist folder should be ignore and should be be checked in to git

Reply:
fixed